src/main/java/at/ac/tuwien/kr/alpha/grounder/NaiveGrounder.java

```diff
@@ -52,8 +52,7 @@
 import static java.util.Collections.singletonList;
 
 /**
- * A semi-naive grounder.
- * Copyright (c) 2016-2018, the Alpha Team.
+ * A semi-naive grounder. Copyright (c) 2016-2018, the Alpha Team.
```

Comment:
If we keep this copyright statement, it should stay in its own line for better readability. We could also drop it, however, since it is redundant to the one in line 2.

Reply:
done (see item below)